### DISCUSSION TITLE
how to enter dark mode in github

### DISCUSSION THREAD

**Discussion by Lagat-emmanuel1** [community/community#121246] [Accessibility]
### Select Topic Area

Question

### Body

how to enter dark mode in github

**Comment by Jv2350** [comment]
Click on your profile picture in the top right corner
From the dropdown menu, select Settings
In the left sidebar, click on Appearance
Under Theme, select Dark from the dropdown menu

**Comment by fdistasio** [comment]
https://github.com/settings/appearance

**Comment by niladri-1** [comment]
# Sure @Lagat39, here's the process:

1. **Sign in**: Go to GitHub and sign in.
2. **Access Settings**: Click on your profile picture in the top-right corner and select "Settings".
3. **Appearance**: On the left sidebar, click on "Appearance".
4. **Dark Mode**: Look for the "Theme" section and choose "Dark".
5. **Save**: Scroll down and click "Save appearance preference".

That's it! You've enabled dark mode on GitHub.

**Comment by cehfisher** [comment]
Hi @Lagat39 - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/121246#discussioncomment-9251737). Could you please confirm and close out this issue? Thank you! (cc @queenofcorgis)
